packages/amplify-graphql-api-construct-tests/src/__tests__/sql-models.test.ts

```diff
@@ -0,0 +1,214 @@
+import * as path from 'path';
+import * as fs from 'fs-extra';
+import {
+  createNewProjectDir,
+  deleteDBInstance,
+  deleteDbConnectionConfig,
+  deleteProjectDir,
+  extractVpcConfigFromDbInstance,
+  setupRDSInstanceAndData,
+  storeDbConnectionConfig,
+} from 'amplify-category-api-e2e-core';
+import generator from 'generate-password';
+import { initCDKProject, cdkDeploy, cdkDestroy } from '../commands';
+import { graphql } from '../graphql-request';
+
+jest.setTimeout(1000 * 60 * 60 /* 1 hour */);
+
+interface DBDetails {
+  endpoint: string;
+  port: number;
+  dbName: string;
+  vpcConfig: {
+    vpcId: string;
+    securityGroupIds: string[];
+    subnetAvailabilityZones: {
+      subnetId: string;
+      availabilityZone: string;
+    }[];
+  };
+  ssmPaths: {
+    hostnameSsmPath: string;
+    portSsmPath: string;
+    usernameSsmPath: string;
+    passwordSsmPath: string;
+    databaseNameSsmPath: string;
+  };
+}
+
+describe('CDK GraphQL Transformer', () => {
+  let projRoot: string;
+  let projFolderName: string;
+
+  const [username, password, identifier] = generator.generateMultiple(3);
+
+  const region = process.env.AWS_REGION ?? 'us-east-1';
+
+  const dbname = 'default_db';
+
+  let dbDetails: DBDetails;
+
+  beforeAll(async () => {
+    dbDetails = await setupDatabase({
+      identifier,
+      engine: 'mysql',
+      dbname,
+      username,
+      password,
+      region,
+    });
+  });
+
+  afterAll(async () => {
+    await cleanupDatabase({ identifier: identifier, region, dbDetails });
+  });
+
+  beforeEach(async () => {
+    projFolderName = 'sqlmodels';
+    projRoot = await createNewProjectDir(projFolderName);
+  });
+
+  afterEach(async () => {
+    try {
+      const result = await cdkDestroy(projRoot, '--all');
+      console.log(`cdkDestroy result: ${JSON.stringify(result)}`);
+    } catch (_) {
+      /* No-op */
+    }
+
+    deleteProjectDir(projRoot);
+  });
+
+  ['2.80.0', 'latest'].forEach((cdkVersion) => {
```

Comment:
Your choice if you want to run both of these versions. We just run the basic smoke test against both to ensure we don't have grosse type errors occuring, etc, but it DOES slow you down quite a bit when you're running these.

Reply:
Spectacular. I'll take out the explicitly versioned one.